payments-core/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -310,6 +313,13 @@ constructor(
             isVoucher = false,
             requiresMandate = false,
             hasDelayedSettlement = false
+        ),
+        USBankAccount(
+            code = "us_bank_account",
+            isReusable = true,
+            isVoucher = false,
+            requiresMandate = true,
```

Comment:
This maps up to a mandate parameter being set on confirmation I believe.  Is that required for this payment method?

Reply:
Yes. There's 3 possibilities:
1. New mandate is created with `mandate_data`
2. Explicit `mandate=` is passed
3. Existing active one already exists and is auto-selected